.github/workflows/publish-code.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish release
+
+on:
+  release:
+    types:
+      - published
+
+env:
+  NODE_VERSION: '10.x'   # Node 10 LTS
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check github.ref starts with 'refs/tags/'
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+        run: |
+          echo Error! github.ref does not start with 'refs/tags'
+          echo github.ref: ${{ github.ref }}
+          exit 1
+      - name: Set version number environment variable
+        env:
+          github_ref: ${{ github.ref }}
+        run: |
+          ref="${github_ref:10}"
+          echo $ref
+          echo "::set-env name=ref::$ref"
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: yarn
+        run: yarn
+      - name: yarn docs2
+        working-directory: docs2
+        run: yarn
+      - name: yarn gatsby build
+        working-directory: docs2
+        run: yarn gatsby build
+      - name: Publish docs artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: Documentation
+          path: docs2/public/**
+          if-no-files-found: error
+      - name: Use .NET Core 3.1 SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
+          source-url: https://api.nuget.org/v3/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.NUGET_AUTH_TOKEN}}
+      - name: Install dependencies
+        working-directory: src
+        run: dotnet restore
+        env:
+          DOTNET_NOLOGO: true
+          DOTNET_CLI_TELEMETRY_OPTOUT: true
+      - name: Build solution (release config)
+        working-directory: src
+        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:Version=$ref
```

Comment:
Now I seem to understand that you were talking about automatic version overwriting when creating a release. In fact, it becomes automatic only after we automate it 😄 . Thus, when creating a release, we are now obliged to give it only a tag in the semver format, right?

Reply:
yup